internal/eacloverlay/doc.go

```diff
@@ -0,0 +1,10 @@
+// Package eacloverlay implements conversion between
+// NeoFS eACL and S3 access policies.
+// Current limitations:
+// 1. Canonical user identifiers are expected to contain public key in hex.
+// 2. Resource identifiers are expected to contain object id in base 58.
```

Comment:
Why don't we use object name instead of object id? If I understand correctly we can add filter by `FileName` attribute. This make sense because bucket policy doesn't apply to a specific version of resources (unlike object ACL)